src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Security/PermissionsController.cs

```diff
@@ -97,6 +97,12 @@ public async Task<ActionResult<ReleaseStatusPermissionsViewModel>> GetReleaseSta
                 })
                 .OrElse(() => new ReleaseStatusPermissionsViewModel());
         }
+        
+        [HttpGet("release/{releaseId}/data/{dataFileName}")]
+        public async Task<ActionResult<DataFilePermissions>> GetDataFilePermissions(Guid releaseId, string dataFileName)
```

Comment:
Just wondering if we really need an endpoint for this. It's quite a lot of additional code across the frontend/backend for something that can be easily handled by the frontend (all of the infrastructure is in place, including the `terminalImportStatuses` in `ImporterStatus`). 

It also means we're doing 2 requests for every single updating file status, which is lot of extra noise that I'm not too keen on.

Reply:
I'd prefer to keep the back end and the front end using the same rules where we're able to.  It means they're both singing from the same hymn sheet.  

Because the permissions call doesn't happen until the actual status has changed (it doesn't happen with every tick of the percentages going up), it'd only happen a maximum of of 4 times for the life of a file import.

Reply:
Cool fair enough. Just had a thought though, why not put it on the actual import status so we can skip making the second request?

Reply:
I think because the "get status" call is the one that gets hammered (it's the one that updates the percentages as well as a status).  While it's probably not a huge deal, this'd mean we'd be doing permission checks many more times than we'd need to be making over the course of a file upload, and when you've got multiple files uploading, that'd add up.  I'd be in favour of keeping it as is for now if that's ok.

Reply:
Fair point, happy to proceed with this!